lib/charms/synapse/v1/matrix_auth.py

```diff
@@ -240,25 +239,14 @@ def encrypt_string(cls, key: bytes, plaintext: str) -> str:
         Returns:
             encrypted text.
         """
-        if not plaintext:
-            return ""
-        # Initialize the cipher with AES-256 and ECB mode
-        cipher = Cipher(algorithms.AES(key), modes.ECB())
-        encryptor = cipher.encryptor()
-
-        # Pad the plaintext to match the block size (16 bytes)
-        padder = PKCS7(algorithms.AES.block_size).padder()
-        padded_data = padder.update(plaintext.get_secret_value().encode()) + padder.finalize()
-
-        # Encrypt the padded plaintext
-        ciphertext = encryptor.update(padded_data) + encryptor.finalize()
-
-        # Return ciphertext as Base64 for easy handling
-        return base64.b64encode(ciphertext).decode()
+        plaintext = cast(SecretStr, plaintext)
+        f = Fernet(key)
```

Comment:
The variable name should probably be more explicit like `encryptor`, `decryptor`